gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc_asyncio.py.j2

```diff
@@ -388,7 +388,7 @@ class {{ service.grpc_asyncio_transport_name }}({{ service.name }}Transport):
         return self._stubs["test_iam_permissions"]
     {% endif %}
 
-    {{ shared_macros.prep_wrapped_messages_async_method(service)|indent(4) }}
```

Comment:
I'm concerned in that higher up in this functions, we add `set_iam_policy` if `opts.add_iam_methods`. But the TODO above `{% if opts.add_iam_methods %}` makes it seem like that may not be used any more. WDYT?

Incidentally, I just filed #2196 to remove these TODOs.

Reply:
You're right, the methods defined within async client aren't used. The ones defined in async mixins file are.